Mathlib/Order/Filter/Basic.lean

```diff
@@ -2165,6 +2165,16 @@ theorem pure_bind (a : α) (m : α → Filter β) : bind (pure a) m = m a := by
   simp only [Bind.bind, bind, map_pure, join_pure]
 #align filter.pure_bind Filter.pure_bind
 
+@[simp]
+theorem map_bind {α β} (m : β → γ) (f : Filter α) (g : α → Filter β) :
```

Comment:
Do we have guidelines about lemmas like these two? Neither of LHS and RHS seem definitely simpler to me.

Reply:
I agree it's probably best not to make them `simp`, I'll remove the tag.